Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -505,13 +500,6 @@ extension Purchases {
         attributionPoster.post(attributionData: data, fromNetwork: network, networkUserId: networkUserId)
     }
 
-    private func postAppleSearchAddsAttributionCollectionIfNeeded() {
-        guard Self.automaticAppleSearchAdsAttributionCollection else {
```

Comment:
This is no longer used, but it's only deprecated. I think that might confuse users because it's not just deprecated; but it will no longer work.

The alternative is to obsolete it, but that would require a new major version. Either way it's breaking behavior though. Should we make the old property "secretly" enable the new one instead?

Reply:
so... we could make `automaticAppleSearchAdsAttributionCollection` enable `attribution.automaticAdServicesAttributionTokenCollection`, but that wouldn't work for devices below 14.3... 

if we want devices <14.3 to work, we would need to add back all of the appleSearchAds stuff.

Reply:
there was a slack discussion that we are okay with cutting support for <14.3. but i understand this is a semver issue...

Reply:
The main thing is how can we do this without making it seem like everything is working for users setting the old property, but then realizing it's not doing anything.

Reply:
@aboedo thoughts on a 5.0.0 obsoleting iad?

Reply:
seeing info coming through for a pre-14.3 user :/ will add all the iad code back.